test/integration/stop/multi_app/instances_enabled/app1/router.init.lua

```diff
@@ -0,0 +1,12 @@
+local inst_name = os.getenv('TARANTOOL_INSTANCE_NAME')
```

Comment:
We have a lot of applications in our integration tests. Let's simply create symbolic links to those that are used.

Reply:
ALso dd it